pinot-core/src/test/java/org/apache/pinot/core/util/SchemaUtilsTest.java

```diff
@@ -140,7 +140,7 @@ public void testCompatibilityWithTableConfig() {
         .addDateTime(TIME_COLUMN, DataType.LONG, "1:MILLISECONDS:EPOCH", "1:HOURS").build();
     tableConfig = new TableConfigBuilder(TableType.REALTIME).setTableName(TABLE_NAME).setTimeColumnName(TIME_COLUMN)
         .setIngestionConfig(
-            new IngestionConfig(null, null, null, Lists.newArrayList(new TransformConfig("colA", "round(colB, 1000)"))))
+            new IngestionConfig(null, null, null, Lists.newArrayList(new TransformConfig("colA", "round(colB, 1000)")), null))
```

Comment:
Many of the lines being modified in this PR are only for adding null at the end of IngestionConfig constructor. Can we create a new IngestionConfig constructor instead?